### PR TITLE
chore(docs): add model registry docs and open-source license

### PR DESCRIPTION
**Pull request by FogDong**
**What this PR does / why we need it**:

Add model registry docs and open-source license

// TODO:
* Add `ormbfile` doc in `ormb` and link it in `klever model registry`.
* Add details of model serving.
* Fulfill the contribute doc.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #44 

**Special notes for your reviewer**:

/cc @simon-cj @gaocegege 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/docs_zh/developers/api.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# API 文档
```
Comment (gaocegege) [inline docs/docs_zh/developers/api.md:1]: 我们有静态 API 文档生成的工具么（for nirvana）

**Review by gaocegege — COMMENTED** [review]
/approve

/assign @simon-cj

File: docs/docs_zh/developers/api.md (hunk 1 above)
Comment (FogDong) [reply docs/docs_zh/developers/api.md:1]: Nirvana 有的，api 部分我再康康

Comment (FogDong) [reply docs/docs_zh/developers/api.md:1]: /docs/api 下就是之前通过 nirvana 生成的

Comment (gaocegege) [reply docs/docs_zh/developers/api.md:1]: 我们可以用 nirvana 直接生成么

**Pull request merged**